src/components/TabButtons.tsx

```diff
@@ -21,7 +21,7 @@ const TabButtons = ({ queries }: TabButtonsProps) => {
       <div className="flex gap-6">
         {queries.map(({ segment, name }, index) => {
           const isTabSelected =
-            pathName.includes(segment) || (urlSegment === null && index === 0);
+            pathName.endsWith(segment) || (urlSegment === null && index === 0);
```

Comment:
This simplifies our logic. If we use `include`, we have to account for overlaps.